### DISCUSSION TITLE
Harmonize arguments in read_csv

### DISCUSSION THREAD

**Discussion by matthiasgomolka** [duckdb/duckdb#13800] [Ideas]
I've discovered that the arguments in `read_csv()` have different names, depending on whether you call it within SQL or from Python directly.

Here is an example to illustrate what I mean:

```python
con.sql("SELECT * FROM read_csv('file.csv', escape='"') # the argument is named 'escape'")

# vs

duckdb.read_csv('file.csv', escape_char='"') # the argument is named 'escape_char'")
```

This inconsistency applies to other arguments as well which caused me some headaches.

From my point of view, it would be best if `duckdb.read_sql()` would also use the arguments as listed here: https://duckdb.org/docs/data/csv/overview#parameters 

But maybe there is some underlying (or obvious) reason I don't see?

**Comment by Tishj** [comment]
The Python `read_csv` method uses the same names as the pandas `read_csv` method where possible

This is intentional

**Reply by matthiasgomolka** [reply]
Thanks for your quick reply! Coming from R I'm not the biggest fan of pandas 😉 so I didn't notice. Nonetheless, this makes sense. 

But perhaps it would be helpful if there was a hint or a somewhat more obvious documentation?
